saleor/order/utils.py

```diff
@@ -333,3 +334,8 @@ def get_voucher_discount_for_order(order: Order) -> Money:
     if order.voucher.type == VoucherType.SPECIFIC_PRODUCT:
         return get_products_voucher_discount_for_order(order.voucher)
     raise NotImplementedError("Unknown discount type")
+
+
+def match_orders_with_new_user(user: User) -> None:
+    """Match Orders with a new user."""
```

Comment:
That line does explain nothing more to me than the func name :) maybe worth changing or reming entirely.